__init__.py

```diff
@@ -163,12 +163,19 @@ def initialize(self):
 
         # Support query for active alarms from other skills
         self.add_event("private.mycroftai.has_alarm", self.on_has_alarm)
+        self.add_event("skill.alarm.query-active", self.handle_active_alarm_query)
 
     def on_has_alarm(self, message):
         """Reply to requests for alarm on/off status."""
         total = len(self.settings["alarm"])
         self.bus.emit(message.response(data={"active_alarms": total}))
 
+    def handle_active_alarm_query(self, _):
+        """Event handler for the skill.alarm.query-active command."""
+        event_data = {"active_alarms": bool(self.settings["alarm"])}
+        event = Message("skill.alarm.active-queried", data=event_data)
+        self.bus.emit(event)
```

Comment:
Is this effectively duplicating the method above `on_has_alarm()` but casting to a boolean rather than providing a number of alarms?

Reply:
I will be removing the on_has_alarm.  I will put a TODO indicating it is obsolete